### PR TITLE
feat/insertion of transition effects in some components.

### PR DESCRIPTION
**Pull request by Hardsys-Andre**
I inserted some transition effects on the Landing Page in order to make the page more attractive.

### CODE REVIEW

**Comment by Hardsys-Andre** [comment]
Hello @Guboehm, Once the adjustments to the effects and the dApp button in the footer were completed, the logo in the header was also changed.

File: src/components/04-templates/HeroSection.tsx [hunk 1]
```diff
@@ -1,12 +1,11 @@
-import { CardHero, TheHeader } from "@/components/02-molecules";
+import { CardHero } from "@/components/02-molecules";
 
 export const HeroSection = () => {
 	return (
 		<section
-			id="home"
+			id=""
```
Comment (Guboehm) [inline src/components/04-templates/HeroSection.tsx]: Is this id really empty?

File: src/components/01-atoms/ui/randomMovingImage.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+					animation: "randomMove 25s linear infinite", // 10 segundos de duração, loop infinito
+				}}
+			>
+				<Image src={bitcoin01} alt={"Bitcoin Icon"} width={40} height={40} />
+			</div>
+			<div
+				className="fixed h-[40px] w-[40px] opacity-70"
+				style={{
+					zIndex: 1000,
+					animation: "randomMove2 25s linear infinite", // 10 segundos de duração, loop infinito
```
Comment (Guboehm) [inline src/components/01-atoms/ui/randomMovingImage.tsx]: remove all comments

**Review by Guboehm — CHANGES_REQUESTED** [review]

**Comment by Hardsys-Andre** [comment]
Hello @Guboehm, removing comments in the randomMovingImage file code and inserting a link in the logo to return to the top of the page.

**Review by Guboehm — APPROVED** [review]

**Pull request merged**